src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -318,11 +347,12 @@ public void validateMedicalInventoryRow(MedicalInventory inventory, List<Medical
 				double theoQty = medicalInventoryRow.getTheoreticQty();
 				if (mainStoreQty != theoQty) {
 					lotUpdated = true;
+					double difference = mainStoreQty - theoQty;
 					medDescriptionForLotUpdated
 						.append("\n")
```

Comment:
in this file all the  string versions of `"\n"` should be replaced with the character version `'\n'`

Reply:
Done, thank you!